{
    "version": "https://jsonfeed.org/version/1",
    "title": "Security Advisory for Python packages hosted at PyPI.org",
    "home_page_url": "https://github.com/advisories?query=type%3Areviewed+ecosystem%3Apip",
    "feed_url": "https://azu.github.io/github-advisory-database-rss/pip.json",
    "description": "Security Advisory for Python packages hosted at PyPI.org on GitHub",
    "items": [
        {
            "content_html": "<p>The <code>ctx</code> hosted project on <a href=\"https://pypi.org/project/ctx/\">PyPI</a> was taken over via user account compromise and replaced with a malicious project which contained runtime code that collected the content of <code>os.environ.items()</code> when instantiating <code>Ctx</code> objects. The captured environment variables were sent as a base64 encoded query parameter to a heroku application running at <code>https://anti-theft-web.herokuapp.com</code>.</p>\n<p>If you installed the package between May 14, 2022 and May 24, 2022, and your environment variables contain sensitive data like passwords and API keys (like <code>AWS_ACCESS_KEY_ID</code> and <code>AWS_SECRET_ACCESS_KEY</code>), we advise you to rotate your passwords and keys, then perform an audit to determine if they were exploited.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://github.com/figlief/ctx\">https://github.com/figlief/ctx</a></li>\n<li><a href=\"https://isc.sans.edu/forums/diary/ctx+Python+Library+Updated+with+Extra+Features/28678/\">https://isc.sans.edu/forums/diary/ctx+Python+Library+Updated+with+Extra+Features/28678/</a></li>\n<li><a href=\"https://portswigger.net/daily-swig/malicious-python-library-ctx-removed-from-pypi-repo\">https://portswigger.net/daily-swig/malicious-python-library-ctx-removed-from-pypi-repo</a></li>\n<li><a href=\"https://python-security.readthedocs.io/pypi-vuln/index-2022-05-24-ctx-domain-takeover.html\">https://python-security.readthedocs.io/pypi-vuln/index-2022-05-24-ctx-domain-takeover.html</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-4g82-3jcr-q52w\">https://github.com/advisories/GHSA-4g82-3jcr-q52w</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-4g82-3jcr-q52w",
            "title": "[ctx] Malware in ctx",
            "date_modified": "2022-05-26T00:01:00.000Z",
            "date_published": "2022-05-25T23:09:55.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-4g82-3jcr-q52w"
            }
        },
        {
            "content_html": "<p>UltraJSON (aka ujson) 1.34 through 5.1.0 has a stack-based buffer overflow in Buffer_AppendIndentUnchecked (called from encode).</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2021-45958\">https://nvd.nist.gov/vuln/detail/CVE-2021-45958</a></li>\n<li><a href=\"https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=36009\">https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=36009</a></li>\n<li><a href=\"https://github.com/google/oss-fuzz-vulns/blob/main/vulns/ujson/OSV-2021-955.yaml\">https://github.com/google/oss-fuzz-vulns/blob/main/vulns/ujson/OSV-2021-955.yaml</a></li>\n<li><a href=\"https://github.com/ultrajson/ultrajson/issues/501\">https://github.com/ultrajson/ultrajson/issues/501</a></li>\n<li><a href=\"https://github.com/ultrajson/ultrajson/issues/502#issuecomment-1031747284\">https://github.com/ultrajson/ultrajson/issues/502#issuecomment-1031747284</a></li>\n<li><a href=\"https://github.com/ultrajson/ultrajson/pull/504\">https://github.com/ultrajson/ultrajson/pull/504</a></li>\n<li><a href=\"https://lists.debian.org/debian-lts-announce/2022/02/msg00023.html\">https://lists.debian.org/debian-lts-announce/2022/02/msg00023.html</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/CN7W3GOXALINKFUUE7ICQIC2EF5HNKUQ/\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/CN7W3GOXALINKFUUE7ICQIC2EF5HNKUQ/</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/O6JUWQTJLA2CMG4CJN7DCUVSOXLZIIXL/\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/O6JUWQTJLA2CMG4CJN7DCUVSOXLZIIXL/</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/ULX35TSWLBBIMEH44MUORPXYYRZKEDC6/\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/ULX35TSWLBBIMEH44MUORPXYYRZKEDC6/</a></li>\n<li><a href=\"https://github.com/ultrajson/ultrajson/pull/519\">https://github.com/ultrajson/ultrajson/pull/519</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-fh56-85cw-5pq6\">https://github.com/advisories/GHSA-fh56-85cw-5pq6</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-fh56-85cw-5pq6",
            "title": "[ujson] Out-of-bounds Write in ujson",
            "date_modified": "2022-05-25T22:11:53.000Z",
            "date_published": "2022-01-21T23:27:45.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-fh56-85cw-5pq6"
            }
        },
        {
            "content_html": "<p>Python package pysaml2 version 4.5.0 and earlier reuses the initialization vector across encryptions in the IDP server, resulting in weak encryption of data.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2017-1000246\">https://nvd.nist.gov/vuln/detail/CVE-2017-1000246</a></li>\n<li><a href=\"https://github.com/rohe/pysaml2/issues/417\">https://github.com/rohe/pysaml2/issues/417</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-cq94-qf6q-mf2h\">https://github.com/advisories/GHSA-cq94-qf6q-mf2h</a></li>\n<li><a href=\"https://github.com/IdentityPython/pysaml2/pull/519/commits/7323f5c20efb59424d853c822e7a26d1aa3e84aa\">https://github.com/IdentityPython/pysaml2/pull/519/commits/7323f5c20efb59424d853c822e7a26d1aa3e84aa</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-cq94-qf6q-mf2h",
            "title": "[pysaml2] Moderate severity vulnerability that affects pysaml2",
            "date_modified": "2022-05-25T21:43:44.000Z",
            "date_published": "2018-07-16T16:50:30.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-cq94-qf6q-mf2h"
            }
        },
        {
            "content_html": "<p>All versions of package url-regex are vulnerable to Regular Expression Denial of Service (ReDoS) which can cause the CPU usage to crash.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2022-21195\">https://nvd.nist.gov/vuln/detail/CVE-2022-21195</a></li>\n<li><a href=\"https://github.com/AlexFlipnote/url_regex/blob/master/url_regex/url_regex.py\">https://github.com/AlexFlipnote/url_regex/blob/master/url_regex/url_regex.py</a></li>\n<li><a href=\"https://snyk.io/vuln/SNYK-PYTHON-URLREGEX-2347643\">https://snyk.io/vuln/SNYK-PYTHON-URLREGEX-2347643</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-hg3w-7hj9-m3f7\">https://github.com/advisories/GHSA-hg3w-7hj9-m3f7</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-hg3w-7hj9-m3f7",
            "title": "[url_regex] Regular expression denial of service in url_regex",
            "date_modified": "2022-05-25T20:35:05.000Z",
            "date_published": "2022-05-21T00:00:25.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-hg3w-7hj9-m3f7"
            }
        },
        {
            "content_html": "<p>Calibre-Web before 0.6.18 allows user table SQL Injection.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2022-30765\">https://nvd.nist.gov/vuln/detail/CVE-2022-30765</a></li>\n<li><a href=\"https://github.com/janeczku/calibre-web/blob/master/SECURITY.md\">https://github.com/janeczku/calibre-web/blob/master/SECURITY.md</a></li>\n<li><a href=\"https://github.com/janeczku/calibre-web/releases/tag/0.6.18\">https://github.com/janeczku/calibre-web/releases/tag/0.6.18</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-8ppf-x4gr-2x7g\">https://github.com/advisories/GHSA-8ppf-x4gr-2x7g</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-8ppf-x4gr-2x7g",
            "title": "[calibreweb] SQL injection in calibreweb",
            "date_modified": "2022-05-25T20:14:11.000Z",
            "date_published": "2022-05-17T00:01:46.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-8ppf-x4gr-2x7g"
            }
        },
        {
            "content_html": "<p>Cross-site Scripting (XSS) - Generic in GitHub repository octoprint/octoprint prior to 1.8.0. The Stream URL of octoprint application allowing a xss payload to execute.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2022-1432\">https://nvd.nist.gov/vuln/detail/CVE-2022-1432</a></li>\n<li><a href=\"https://github.com/octoprint/octoprint/commit/6d259d7e6f5b0de9a1c762831537a386e53978d3\">https://github.com/octoprint/octoprint/commit/6d259d7e6f5b0de9a1c762831537a386e53978d3</a></li>\n<li><a href=\"https://huntr.dev/bounties/cb545c63-a3c1-4d57-8f06-e4593ab389bf\">https://huntr.dev/bounties/cb545c63-a3c1-4d57-8f06-e4593ab389bf</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-h8pc-j334-jjhm\">https://github.com/advisories/GHSA-h8pc-j334-jjhm</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-h8pc-j334-jjhm",
            "title": "[OctoPrint] Cross-site Scripting in OctoPrint",
            "date_modified": "2022-05-25T19:44:28.000Z",
            "date_published": "2022-05-19T00:00:30.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-h8pc-j334-jjhm"
            }
        },
        {
            "content_html": "<p>Cross-site Scripting (XSS) - DOM in GitHub repository octoprint/octoprint prior to 1.8.0. The login endpoint allows for javascript injection which may lead to account takeover in a phishing scenario.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2022-1430\">https://nvd.nist.gov/vuln/detail/CVE-2022-1430</a></li>\n<li><a href=\"https://github.com/octoprint/octoprint/commit/8087528e4a7ddd15c7d95ff662deb5ef7de90045\">https://github.com/octoprint/octoprint/commit/8087528e4a7ddd15c7d95ff662deb5ef7de90045</a></li>\n<li><a href=\"https://huntr.dev/bounties/0cd30d71-1e32-4a0b-b4c3-faaa1907b541\">https://huntr.dev/bounties/0cd30d71-1e32-4a0b-b4c3-faaa1907b541</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-x7r7-wmj8-vv5g\">https://github.com/advisories/GHSA-x7r7-wmj8-vv5g</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-x7r7-wmj8-vv5g",
            "title": "[OctoPrint] Cross-site Scripting in OctoPrint",
            "date_modified": "2022-05-25T19:44:09.000Z",
            "date_published": "2022-05-19T00:00:31.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-x7r7-wmj8-vv5g"
            }
        },
        {
            "content_html": "<p>ADMesh through 0.98.4 has a heap-based buffer over-read in stl_update_connects_remove_1 (called from stl_remove_degenerate) in connect.c in libadmesh.a.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2018-25033\">https://nvd.nist.gov/vuln/detail/CVE-2018-25033</a></li>\n<li><a href=\"https://github.com/admesh/admesh/issues/28\">https://github.com/admesh/admesh/issues/28</a></li>\n<li><a href=\"https://lists.debian.org/debian-lts-announce/2022/05/msg00029.html\">https://lists.debian.org/debian-lts-announce/2022/05/msg00029.html</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-5jrq-582v-9767\">https://github.com/advisories/GHSA-5jrq-582v-9767</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-5jrq-582v-9767",
            "title": "[admesh] Out-of-bounds read in admesh",
            "date_modified": "2022-05-25T19:38:06.000Z",
            "date_published": "2022-05-09T00:00:51.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-5jrq-582v-9767"
            }
        },
        {
            "content_html": "<p>Apache Superset before 1.4.2 is vulnerable to SQL injection in chart data requests. Users should update to 1.4.2 or higher which addresses this issue.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2022-27479\">https://nvd.nist.gov/vuln/detail/CVE-2022-27479</a></li>\n<li><a href=\"https://lists.apache.org/thread/94th50j5d0y2fw7ysx0g7w3t6jk3z7q6\">https://lists.apache.org/thread/94th50j5d0y2fw7ysx0g7w3t6jk3z7q6</a></li>\n<li><a href=\"https://lists.apache.org/thread/ztb9b6jd9rngoxwvq8r4fhpp401o613y\">https://lists.apache.org/thread/ztb9b6jd9rngoxwvq8r4fhpp401o613y</a></li>\n<li><a href=\"http://www.openwall.com/lists/oss-security/2022/04/13/3\">http://www.openwall.com/lists/oss-security/2022/04/13/3</a></li>\n<li><a href=\"https://github.com/pypa/advisory-database/tree/main/vulns/apache-superset/PYSEC-2022-188.yaml\">https://github.com/pypa/advisory-database/tree/main/vulns/apache-superset/PYSEC-2022-188.yaml</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-wh73-hpcg-v32j\">https://github.com/advisories/GHSA-wh73-hpcg-v32j</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-wh73-hpcg-v32j",
            "title": "[apache-superset] SQL injection in apache-superset",
            "date_modified": "2022-05-25T19:34:58.000Z",
            "date_published": "2022-04-14T00:00:14.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-wh73-hpcg-v32j"
            }
        },
        {
            "content_html": "<h3 id=\"impact\">Impact</h3>\n<p>The implementation of depthwise ops in TensorFlow is vulnerable to a denial of service via <code>CHECK</code>-failure (assertion failure) caused by overflowing the number of elements in a tensor:</p>\n<pre><code class=\"language-python\">import tensorflow as tf\n\ninput = tf.constant(1, shape=[1, 4, 4, 3], dtype=tf.float32)\nfilter_sizes = tf.constant(1879048192, shape=[13], dtype=tf.int32)\nout_backprop = tf.constant(1, shape=[1, 4, 4, 3], dtype=tf.float32)\ntf.raw_ops.DepthwiseConv2dNativeBackpropFilter(\n    input=input, filter_sizes=filter_sizes, out_backprop=out_backprop, strides=[1, 1, 1, 1], padding=\"SAME\")\n</code></pre>\n<p>This is another instance of <a href=\"https://github.com/tensorflow/tensorflow/blob/master/tensorflow/security/advisory/tfsa-2021-198.md\">TFSA-2021-198</a> (CVE-2021-41197).</p>\n<h3 id=\"patches\">Patches</h3>\n<p>We have patched the issue in GitHub commit <a href=\"https://github.com/tensorflow/tensorflow/commit/3796cc4fcd93ae55812a457abc96dcd55fbb854b\">3796cc4fcd93ae55812a457abc96dcd55fbb854b</a>.</p>\n<p>The fix will be included in TensorFlow 2.9.0. We will also cherrypick this commit on TensorFlow 2.8.1, TensorFlow 2.7.2, and TensorFlow 2.6.4, as these are also affected and still in supported range.</p>\n<h3 id=\"for-more-information\">For more information</h3>\n<p>Please consult <a href=\"https://github.com/tensorflow/tensorflow/blob/master/SECURITY.md\">our security guide</a> for more information regarding the security model and how to contact us with issues and questions.</p>\n<h3 id=\"attribution\">Attribution</h3>\n<p>This vulnerability has been reported by Neophytos Christou from Secure Systems Lab at Brown University.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://github.com/tensorflow/tensorflow/security/advisories/GHSA-mw6j-hh29-h379\">https://github.com/tensorflow/tensorflow/security/advisories/GHSA-mw6j-hh29-h379</a></li>\n<li><a href=\"https://github.com/tensorflow/tensorflow/commit/3796cc4fcd93ae55812a457abc96dcd55fbb854b\">https://github.com/tensorflow/tensorflow/commit/3796cc4fcd93ae55812a457abc96dcd55fbb854b</a></li>\n<li><a href=\"https://github.com/tensorflow/tensorflow/blob/master/tensorflow/security/advisory/tfsa-2021-198.md\">https://github.com/tensorflow/tensorflow/blob/master/tensorflow/security/advisory/tfsa-2021-198.md</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-mw6j-hh29-h379\">https://github.com/advisories/GHSA-mw6j-hh29-h379</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-mw6j-hh29-h379",
            "title": "[tensorflow-gpu] `CHECK` failure in depthwise ops via overflows",
            "date_modified": "2022-05-25T19:33:51.000Z",
            "date_published": "2022-05-25T19:33:50.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-mw6j-hh29-h379"
            }
        },
        {
            "content_html": "<h3 id=\"impact\">Impact</h3>\n<p>The implementation of depthwise ops in TensorFlow is vulnerable to a denial of service via <code>CHECK</code>-failure (assertion failure) caused by overflowing the number of elements in a tensor:</p>\n<pre><code class=\"language-python\">import tensorflow as tf\n\ninput = tf.constant(1, shape=[1, 4, 4, 3], dtype=tf.float32)\nfilter_sizes = tf.constant(1879048192, shape=[13], dtype=tf.int32)\nout_backprop = tf.constant(1, shape=[1, 4, 4, 3], dtype=tf.float32)\ntf.raw_ops.DepthwiseConv2dNativeBackpropFilter(\n    input=input, filter_sizes=filter_sizes, out_backprop=out_backprop, strides=[1, 1, 1, 1], padding=\"SAME\")\n</code></pre>\n<p>This is another instance of <a href=\"https://github.com/tensorflow/tensorflow/blob/master/tensorflow/security/advisory/tfsa-2021-198.md\">TFSA-2021-198</a> (CVE-2021-41197).</p>\n<h3 id=\"patches\">Patches</h3>\n<p>We have patched the issue in GitHub commit <a href=\"https://github.com/tensorflow/tensorflow/commit/3796cc4fcd93ae55812a457abc96dcd55fbb854b\">3796cc4fcd93ae55812a457abc96dcd55fbb854b</a>.</p>\n<p>The fix will be included in TensorFlow 2.9.0. We will also cherrypick this commit on TensorFlow 2.8.1, TensorFlow 2.7.2, and TensorFlow 2.6.4, as these are also affected and still in supported range.</p>\n<h3 id=\"for-more-information\">For more information</h3>\n<p>Please consult <a href=\"https://github.com/tensorflow/tensorflow/blob/master/SECURITY.md\">our security guide</a> for more information regarding the security model and how to contact us with issues and questions.</p>\n<h3 id=\"attribution\">Attribution</h3>\n<p>This vulnerability has been reported by Neophytos Christou from Secure Systems Lab at Brown University.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://github.com/tensorflow/tensorflow/security/advisories/GHSA-mw6j-hh29-h379\">https://github.com/tensorflow/tensorflow/security/advisories/GHSA-mw6j-hh29-h379</a></li>\n<li><a href=\"https://github.com/tensorflow/tensorflow/commit/3796cc4fcd93ae55812a457abc96dcd55fbb854b\">https://github.com/tensorflow/tensorflow/commit/3796cc4fcd93ae55812a457abc96dcd55fbb854b</a></li>\n<li><a href=\"https://github.com/tensorflow/tensorflow/blob/master/tensorflow/security/advisory/tfsa-2021-198.md\">https://github.com/tensorflow/tensorflow/blob/master/tensorflow/security/advisory/tfsa-2021-198.md</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-mw6j-hh29-h379\">https://github.com/advisories/GHSA-mw6j-hh29-h379</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-mw6j-hh29-h379",
            "title": "[tensorflow-gpu] `CHECK` failure in depthwise ops via overflows",
            "date_modified": "2022-05-25T19:33:51.000Z",
            "date_published": "2022-05-25T19:33:50.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-mw6j-hh29-h379"
            }
        },
        {
            "content_html": "<h3 id=\"impact\">Impact</h3>\n<p>The implementation of depthwise ops in TensorFlow is vulnerable to a denial of service via <code>CHECK</code>-failure (assertion failure) caused by overflowing the number of elements in a tensor:</p>\n<pre><code class=\"language-python\">import tensorflow as tf\n\ninput = tf.constant(1, shape=[1, 4, 4, 3], dtype=tf.float32)\nfilter_sizes = tf.constant(1879048192, shape=[13], dtype=tf.int32)\nout_backprop = tf.constant(1, shape=[1, 4, 4, 3], dtype=tf.float32)\ntf.raw_ops.DepthwiseConv2dNativeBackpropFilter(\n    input=input, filter_sizes=filter_sizes, out_backprop=out_backprop, strides=[1, 1, 1, 1], padding=\"SAME\")\n</code></pre>\n<p>This is another instance of <a href=\"https://github.com/tensorflow/tensorflow/blob/master/tensorflow/security/advisory/tfsa-2021-198.md\">TFSA-2021-198</a> (CVE-2021-41197).</p>\n<h3 id=\"patches\">Patches</h3>\n<p>We have patched the issue in GitHub commit <a href=\"https://github.com/tensorflow/tensorflow/commit/3796cc4fcd93ae55812a457abc96dcd55fbb854b\">3796cc4fcd93ae55812a457abc96dcd55fbb854b</a>.</p>\n<p>The fix will be included in TensorFlow 2.9.0. We will also cherrypick this commit on TensorFlow 2.8.1, TensorFlow 2.7.2, and TensorFlow 2.6.4, as these are also affected and still in supported range.</p>\n<h3 id=\"for-more-information\">For more information</h3>\n<p>Please consult <a href=\"https://github.com/tensorflow/tensorflow/blob/master/SECURITY.md\">our security guide</a> for more information regarding the security model and how to contact us with issues and questions.</p>\n<h3 id=\"attribution\">Attribution</h3>\n<p>This vulnerability has been reported by Neophytos Christou from Secure Systems Lab at Brown University.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://github.com/tensorflow/tensorflow/security/advisories/GHSA-mw6j-hh29-h379\">https://github.com/tensorflow/tensorflow/security/advisories/GHSA-mw6j-hh29-h379</a></li>\n<li><a href=\"https://github.com/tensorflow/tensorflow/commit/3796cc4fcd93ae55812a457abc96dcd55fbb854b\">https://github.com/tensorflow/tensorflow/commit/3796cc4fcd93ae55812a457abc96dcd55fbb854b</a></li>\n<li><a href=\"https://github.com/tensorflow/tensorflow/blob/master/tensorflow/security/advisory/tfsa-2021-198.md\">https://github.com/tensorflow/tensorflow/blob/master/tensorflow/security/advisory/tfsa-2021-198.md</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-mw6j-hh29-h379\">https://github.com/advisories/GHSA-mw6j-hh29-h379</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-mw6j-hh29-h379",
            "title": "[tensorflow-gpu] `CHECK` failure in depthwise ops via overflows",
            "date_modified": "2022-05-25T19:33:51.000Z",
            "date_published": "2022-05-25T19:33:50.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-mw6j-hh29-h379"
            }
        },
        {
            "content_html": "<h3 id=\"impact\">Impact</h3>\n<p>The implementation of depthwise ops in TensorFlow is vulnerable to a denial of service via <code>CHECK</code>-failure (assertion failure) caused by overflowing the number of elements in a tensor:</p>\n<pre><code class=\"language-python\">import tensorflow as tf\n\ninput = tf.constant(1, shape=[1, 4, 4, 3], dtype=tf.float32)\nfilter_sizes = tf.constant(1879048192, shape=[13], dtype=tf.int32)\nout_backprop = tf.constant(1, shape=[1, 4, 4, 3], dtype=tf.float32)\ntf.raw_ops.DepthwiseConv2dNativeBackpropFilter(\n    input=input, filter_sizes=filter_sizes, out_backprop=out_backprop, strides=[1, 1, 1, 1], padding=\"SAME\")\n</code></pre>\n<p>This is another instance of <a href=\"https://github.com/tensorflow/tensorflow/blob/master/tensorflow/security/advisory/tfsa-2021-198.md\">TFSA-2021-198</a> (CVE-2021-41197).</p>\n<h3 id=\"patches\">Patches</h3>\n<p>We have patched the issue in GitHub commit <a href=\"https://github.com/tensorflow/tensorflow/commit/3796cc4fcd93ae55812a457abc96dcd55fbb854b\">3796cc4fcd93ae55812a457abc96dcd55fbb854b</a>.</p>\n<p>The fix will be included in TensorFlow 2.9.0. We will also cherrypick this commit on TensorFlow 2.8.1, TensorFlow 2.7.2, and TensorFlow 2.6.4, as these are also affected and still in supported range.</p>\n<h3 id=\"for-more-information\">For more information</h3>\n<p>Please consult <a href=\"https://github.com/tensorflow/tensorflow/blob/master/SECURITY.md\">our security guide</a> for more information regarding the security model and how to contact us with issues and questions.</p>\n<h3 id=\"attribution\">Attribution</h3>\n<p>This vulnerability has been reported by Neophytos Christou from Secure Systems Lab at Brown University.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://github.com/tensorflow/tensorflow/security/advisories/GHSA-mw6j-hh29-h379\">https://github.com/tensorflow/tensorflow/security/advisories/GHSA-mw6j-hh29-h379</a></li>\n<li><a href=\"https://github.com/tensorflow/tensorflow/commit/3796cc4fcd93ae55812a457abc96dcd55fbb854b\">https://github.com/tensorflow/tensorflow/commit/3796cc4fcd93ae55812a457abc96dcd55fbb854b</a></li>\n<li><a href=\"https://github.com/tensorflow/tensorflow/blob/master/tensorflow/security/advisory/tfsa-2021-198.md\">https://github.com/tensorflow/tensorflow/blob/master/tensorflow/security/advisory/tfsa-2021-198.md</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-mw6j-hh29-h379\">https://github.com/advisories/GHSA-mw6j-hh29-h379</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-mw6j-hh29-h379",
            "title": "[tensorflow-cpu] `CHECK` failure in depthwise ops via overflows",
            "date_modified": "2022-05-25T19:33:51.000Z",
            "date_published": "2022-05-25T19:33:50.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-mw6j-hh29-h379"
            }
        },
        {
            "content_html": "<h3 id=\"impact\">Impact</h3>\n<p>The implementation of depthwise ops in TensorFlow is vulnerable to a denial of service via <code>CHECK</code>-failure (assertion failure) caused by overflowing the number of elements in a tensor:</p>\n<pre><code class=\"language-python\">import tensorflow as tf\n\ninput = tf.constant(1, shape=[1, 4, 4, 3], dtype=tf.float32)\nfilter_sizes = tf.constant(1879048192, shape=[13], dtype=tf.int32)\nout_backprop = tf.constant(1, shape=[1, 4, 4, 3], dtype=tf.float32)\ntf.raw_ops.DepthwiseConv2dNativeBackpropFilter(\n    input=input, filter_sizes=filter_sizes, out_backprop=out_backprop, strides=[1, 1, 1, 1], padding=\"SAME\")\n</code></pre>\n<p>This is another instance of <a href=\"https://github.com/tensorflow/tensorflow/blob/master/tensorflow/security/advisory/tfsa-2021-198.md\">TFSA-2021-198</a> (CVE-2021-41197).</p>\n<h3 id=\"patches\">Patches</h3>\n<p>We have patched the issue in GitHub commit <a href=\"https://github.com/tensorflow/tensorflow/commit/3796cc4fcd93ae55812a457abc96dcd55fbb854b\">3796cc4fcd93ae55812a457abc96dcd55fbb854b</a>.</p>\n<p>The fix will be included in TensorFlow 2.9.0. We will also cherrypick this commit on TensorFlow 2.8.1, TensorFlow 2.7.2, and TensorFlow 2.6.4, as these are also affected and still in supported range.</p>\n<h3 id=\"for-more-information\">For more information</h3>\n<p>Please consult <a href=\"https://github.com/tensorflow/tensorflow/blob/master/SECURITY.md\">our security guide</a> for more information regarding the security model and how to contact us with issues and questions.</p>\n<h3 id=\"attribution\">Attribution</h3>\n<p>This vulnerability has been reported by Neophytos Christou from Secure Systems Lab at Brown University.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://github.com/tensorflow/tensorflow/security/advisories/GHSA-mw6j-hh29-h379\">https://github.com/tensorflow/tensorflow/security/advisories/GHSA-mw6j-hh29-h379</a></li>\n<li><a href=\"https://github.com/tensorflow/tensorflow/commit/3796cc4fcd93ae55812a457abc96dcd55fbb854b\">https://github.com/tensorflow/tensorflow/commit/3796cc4fcd93ae55812a457abc96dcd55fbb854b</a></li>\n<li><a href=\"https://github.com/tensorflow/tensorflow/blob/master/tensorflow/security/advisory/tfsa-2021-198.md\">https://github.com/tensorflow/tensorflow/blob/master/tensorflow/security/advisory/tfsa-2021-198.md</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-mw6j-hh29-h379\">https://github.com/advisories/GHSA-mw6j-hh29-h379</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-mw6j-hh29-h379",
            "title": "[tensorflow-cpu] `CHECK` failure in depthwise ops via overflows",
            "date_modified": "2022-05-25T19:33:51.000Z",
            "date_published": "2022-05-25T19:33:50.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-mw6j-hh29-h379"
            }
        },
        {
            "content_html": "<h3 id=\"impact\">Impact</h3>\n<p>The implementation of depthwise ops in TensorFlow is vulnerable to a denial of service via <code>CHECK</code>-failure (assertion failure) caused by overflowing the number of elements in a tensor:</p>\n<pre><code class=\"language-python\">import tensorflow as tf\n\ninput = tf.constant(1, shape=[1, 4, 4, 3], dtype=tf.float32)\nfilter_sizes = tf.constant(1879048192, shape=[13], dtype=tf.int32)\nout_backprop = tf.constant(1, shape=[1, 4, 4, 3], dtype=tf.float32)\ntf.raw_ops.DepthwiseConv2dNativeBackpropFilter(\n    input=input, filter_sizes=filter_sizes, out_backprop=out_backprop, strides=[1, 1, 1, 1], padding=\"SAME\")\n</code></pre>\n<p>This is another instance of <a href=\"https://github.com/tensorflow/tensorflow/blob/master/tensorflow/security/advisory/tfsa-2021-198.md\">TFSA-2021-198</a> (CVE-2021-41197).</p>\n<h3 id=\"patches\">Patches</h3>\n<p>We have patched the issue in GitHub commit <a href=\"https://github.com/tensorflow/tensorflow/commit/3796cc4fcd93ae55812a457abc96dcd55fbb854b\">3796cc4fcd93ae55812a457abc96dcd55fbb854b</a>.</p>\n<p>The fix will be included in TensorFlow 2.9.0. We will also cherrypick this commit on TensorFlow 2.8.1, TensorFlow 2.7.2, and TensorFlow 2.6.4, as these are also affected and still in supported range.</p>\n<h3 id=\"for-more-information\">For more information</h3>\n<p>Please consult <a href=\"https://github.com/tensorflow/tensorflow/blob/master/SECURITY.md\">our security guide</a> for more information regarding the security model and how to contact us with issues and questions.</p>\n<h3 id=\"attribution\">Attribution</h3>\n<p>This vulnerability has been reported by Neophytos Christou from Secure Systems Lab at Brown University.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://github.com/tensorflow/tensorflow/security/advisories/GHSA-mw6j-hh29-h379\">https://github.com/tensorflow/tensorflow/security/advisories/GHSA-mw6j-hh29-h379</a></li>\n<li><a href=\"https://github.com/tensorflow/tensorflow/commit/3796cc4fcd93ae55812a457abc96dcd55fbb854b\">https://github.com/tensorflow/tensorflow/commit/3796cc4fcd93ae55812a457abc96dcd55fbb854b</a></li>\n<li><a href=\"https://github.com/tensorflow/tensorflow/blob/master/tensorflow/security/advisory/tfsa-2021-198.md\">https://github.com/tensorflow/tensorflow/blob/master/tensorflow/security/advisory/tfsa-2021-198.md</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-mw6j-hh29-h379\">https://github.com/advisories/GHSA-mw6j-hh29-h379</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-mw6j-hh29-h379",
            "title": "[tensorflow-cpu] `CHECK` failure in depthwise ops via overflows",
            "date_modified": "2022-05-25T19:33:51.000Z",
            "date_published": "2022-05-25T19:33:50.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-mw6j-hh29-h379"
            }
        },
        {
            "content_html": "<h3 id=\"impact\">Impact</h3>\n<p>The implementation of depthwise ops in TensorFlow is vulnerable to a denial of service via <code>CHECK</code>-failure (assertion failure) caused by overflowing the number of elements in a tensor:</p>\n<pre><code class=\"language-python\">import tensorflow as tf\n\ninput = tf.constant(1, shape=[1, 4, 4, 3], dtype=tf.float32)\nfilter_sizes = tf.constant(1879048192, shape=[13], dtype=tf.int32)\nout_backprop = tf.constant(1, shape=[1, 4, 4, 3], dtype=tf.float32)\ntf.raw_ops.DepthwiseConv2dNativeBackpropFilter(\n    input=input, filter_sizes=filter_sizes, out_backprop=out_backprop, strides=[1, 1, 1, 1], padding=\"SAME\")\n</code></pre>\n<p>This is another instance of <a href=\"https://github.com/tensorflow/tensorflow/blob/master/tensorflow/security/advisory/tfsa-2021-198.md\">TFSA-2021-198</a> (CVE-2021-41197).</p>\n<h3 id=\"patches\">Patches</h3>\n<p>We have patched the issue in GitHub commit <a href=\"https://github.com/tensorflow/tensorflow/commit/3796cc4fcd93ae55812a457abc96dcd55fbb854b\">3796cc4fcd93ae55812a457abc96dcd55fbb854b</a>.</p>\n<p>The fix will be included in TensorFlow 2.9.0. We will also cherrypick this commit on TensorFlow 2.8.1, TensorFlow 2.7.2, and TensorFlow 2.6.4, as these are also affected and still in supported range.</p>\n<h3 id=\"for-more-information\">For more information</h3>\n<p>Please consult <a href=\"https://github.com/tensorflow/tensorflow/blob/master/SECURITY.md\">our security guide</a> for more information regarding the security model and how to contact us with issues and questions.</p>\n<h3 id=\"attribution\">Attribution</h3>\n<p>This vulnerability has been reported by Neophytos Christou from Secure Systems Lab at Brown University.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://github.com/tensorflow/tensorflow/security/advisories/GHSA-mw6j-hh29-h379\">https://github.com/tensorflow/tensorflow/security/advisories/GHSA-mw6j-hh29-h379</a></li>\n<li><a href=\"https://github.com/tensorflow/tensorflow/commit/3796cc4fcd93ae55812a457abc96dcd55fbb854b\">https://github.com/tensorflow/tensorflow/commit/3796cc4fcd93ae55812a457abc96dcd55fbb854b</a></li>\n<li><a href=\"https://github.com/tensorflow/tensorflow/blob/master/tensorflow/security/advisory/tfsa-2021-198.md\">https://github.com/tensorflow/tensorflow/blob/master/tensorflow/security/advisory/tfsa-2021-198.md</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-mw6j-hh29-h379\">https://github.com/advisories/GHSA-mw6j-hh29-h379</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-mw6j-hh29-h379",
            "title": "[tensorflow] `CHECK` failure in depthwise ops via overflows",
            "date_modified": "2022-05-25T19:33:51.000Z",
            "date_published": "2022-05-25T19:33:50.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-mw6j-hh29-h379"
            }
        },
        {
            "content_html": "<h3 id=\"impact\">Impact</h3>\n<p>The implementation of depthwise ops in TensorFlow is vulnerable to a denial of service via <code>CHECK</code>-failure (assertion failure) caused by overflowing the number of elements in a tensor:</p>\n<pre><code class=\"language-python\">import tensorflow as tf\n\ninput = tf.constant(1, shape=[1, 4, 4, 3], dtype=tf.float32)\nfilter_sizes = tf.constant(1879048192, shape=[13], dtype=tf.int32)\nout_backprop = tf.constant(1, shape=[1, 4, 4, 3], dtype=tf.float32)\ntf.raw_ops.DepthwiseConv2dNativeBackpropFilter(\n    input=input, filter_sizes=filter_sizes, out_backprop=out_backprop, strides=[1, 1, 1, 1], padding=\"SAME\")\n</code></pre>\n<p>This is another instance of <a href=\"https://github.com/tensorflow/tensorflow/blob/master/tensorflow/security/advisory/tfsa-2021-198.md\">TFSA-2021-198</a> (CVE-2021-41197).</p>\n<h3 id=\"patches\">Patches</h3>\n<p>We have patched the issue in GitHub commit <a href=\"https://github.com/tensorflow/tensorflow/commit/3796cc4fcd93ae55812a457abc96dcd55fbb854b\">3796cc4fcd93ae55812a457abc96dcd55fbb854b</a>.</p>\n<p>The fix will be included in TensorFlow 2.9.0. We will also cherrypick this commit on TensorFlow 2.8.1, TensorFlow 2.7.2, and TensorFlow 2.6.4, as these are also affected and still in supported range.</p>\n<h3 id=\"for-more-information\">For more information</h3>\n<p>Please consult <a href=\"https://github.com/tensorflow/tensorflow/blob/master/SECURITY.md\">our security guide</a> for more information regarding the security model and how to contact us with issues and questions.</p>\n<h3 id=\"attribution\">Attribution</h3>\n<p>This vulnerability has been reported by Neophytos Christou from Secure Systems Lab at Brown University.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://github.com/tensorflow/tensorflow/security/advisories/GHSA-mw6j-hh29-h379\">https://github.com/tensorflow/tensorflow/security/advisories/GHSA-mw6j-hh29-h379</a></li>\n<li><a href=\"https://github.com/tensorflow/tensorflow/commit/3796cc4fcd93ae55812a457abc96dcd55fbb854b\">https://github.com/tensorflow/tensorflow/commit/3796cc4fcd93ae55812a457abc96dcd55fbb854b</a></li>\n<li><a href=\"https://github.com/tensorflow/tensorflow/blob/master/tensorflow/security/advisory/tfsa-2021-198.md\">https://github.com/tensorflow/tensorflow/blob/master/tensorflow/security/advisory/tfsa-2021-198.md</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-mw6j-hh29-h379\">https://github.com/advisories/GHSA-mw6j-hh29-h379</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-mw6j-hh29-h379",
            "title": "[tensorflow] `CHECK` failure in depthwise ops via overflows",
            "date_modified": "2022-05-25T19:33:51.000Z",
            "date_published": "2022-05-25T19:33:50.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-mw6j-hh29-h379"
            }
        },
        {
            "content_html": "<h3 id=\"impact\">Impact</h3>\n<p>The implementation of depthwise ops in TensorFlow is vulnerable to a denial of service via <code>CHECK</code>-failure (assertion failure) caused by overflowing the number of elements in a tensor:</p>\n<pre><code class=\"language-python\">import tensorflow as tf\n\ninput = tf.constant(1, shape=[1, 4, 4, 3], dtype=tf.float32)\nfilter_sizes = tf.constant(1879048192, shape=[13], dtype=tf.int32)\nout_backprop = tf.constant(1, shape=[1, 4, 4, 3], dtype=tf.float32)\ntf.raw_ops.DepthwiseConv2dNativeBackpropFilter(\n    input=input, filter_sizes=filter_sizes, out_backprop=out_backprop, strides=[1, 1, 1, 1], padding=\"SAME\")\n</code></pre>\n<p>This is another instance of <a href=\"https://github.com/tensorflow/tensorflow/blob/master/tensorflow/security/advisory/tfsa-2021-198.md\">TFSA-2021-198</a> (CVE-2021-41197).</p>\n<h3 id=\"patches\">Patches</h3>\n<p>We have patched the issue in GitHub commit <a href=\"https://github.com/tensorflow/tensorflow/commit/3796cc4fcd93ae55812a457abc96dcd55fbb854b\">3796cc4fcd93ae55812a457abc96dcd55fbb854b</a>.</p>\n<p>The fix will be included in TensorFlow 2.9.0. We will also cherrypick this commit on TensorFlow 2.8.1, TensorFlow 2.7.2, and TensorFlow 2.6.4, as these are also affected and still in supported range.</p>\n<h3 id=\"for-more-information\">For more information</h3>\n<p>Please consult <a href=\"https://github.com/tensorflow/tensorflow/blob/master/SECURITY.md\">our security guide</a> for more information regarding the security model and how to contact us with issues and questions.</p>\n<h3 id=\"attribution\">Attribution</h3>\n<p>This vulnerability has been reported by Neophytos Christou from Secure Systems Lab at Brown University.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://github.com/tensorflow/tensorflow/security/advisories/GHSA-mw6j-hh29-h379\">https://github.com/tensorflow/tensorflow/security/advisories/GHSA-mw6j-hh29-h379</a></li>\n<li><a href=\"https://github.com/tensorflow/tensorflow/commit/3796cc4fcd93ae55812a457abc96dcd55fbb854b\">https://github.com/tensorflow/tensorflow/commit/3796cc4fcd93ae55812a457abc96dcd55fbb854b</a></li>\n<li><a href=\"https://github.com/tensorflow/tensorflow/blob/master/tensorflow/security/advisory/tfsa-2021-198.md\">https://github.com/tensorflow/tensorflow/blob/master/tensorflow/security/advisory/tfsa-2021-198.md</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-mw6j-hh29-h379\">https://github.com/advisories/GHSA-mw6j-hh29-h379</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-mw6j-hh29-h379",
            "title": "[tensorflow] `CHECK` failure in depthwise ops via overflows",
            "date_modified": "2022-05-25T19:33:51.000Z",
            "date_published": "2022-05-25T19:33:50.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-mw6j-hh29-h379"
            }
        },
        {
            "content_html": "<p>A flaw was found in the Ansible Engine 2.9.18, where sensitive info is not masked by default and is not protected by the no_log feature when using the sub-option feature of the basic.py module. This flaw allows an attacker to obtain sensitive information. The highest threat from this vulnerability is to confidentiality.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2021-20228\">https://nvd.nist.gov/vuln/detail/CVE-2021-20228</a></li>\n<li><a href=\"https://github.com/ansible/ansible/pull/73487\">https://github.com/ansible/ansible/pull/73487</a></li>\n<li><a href=\"https://github.com/ansible/ansible/pull/73492\">https://github.com/ansible/ansible/pull/73492</a></li>\n<li><a href=\"https://github.com/ansible/ansible/pull/73493\">https://github.com/ansible/ansible/pull/73493</a></li>\n<li><a href=\"https://github.com/ansible/ansible/pull/73494\">https://github.com/ansible/ansible/pull/73494</a></li>\n<li><a href=\"https://bugzilla.redhat.com/show_bug.cgi?id=1925002\">https://bugzilla.redhat.com/show_bug.cgi?id=1925002</a></li>\n<li><a href=\"https://www.debian.org/security/2021/dsa-4950\">https://www.debian.org/security/2021/dsa-4950</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-5rrg-rr89-x9mv\">https://github.com/advisories/GHSA-5rrg-rr89-x9mv</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-5rrg-rr89-x9mv",
            "title": "[ansible] Exposure of Sensitive Information in ansible",
            "date_modified": "2022-05-25T19:22:36.000Z",
            "date_published": "2022-05-25T19:22:34.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-5rrg-rr89-x9mv"
            }
        },
        {
            "content_html": "<p>A flaw was found in the Ansible Engine 2.9.18, where sensitive info is not masked by default and is not protected by the no_log feature when using the sub-option feature of the basic.py module. This flaw allows an attacker to obtain sensitive information. The highest threat from this vulnerability is to confidentiality.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2021-20228\">https://nvd.nist.gov/vuln/detail/CVE-2021-20228</a></li>\n<li><a href=\"https://github.com/ansible/ansible/pull/73487\">https://github.com/ansible/ansible/pull/73487</a></li>\n<li><a href=\"https://github.com/ansible/ansible/pull/73492\">https://github.com/ansible/ansible/pull/73492</a></li>\n<li><a href=\"https://github.com/ansible/ansible/pull/73493\">https://github.com/ansible/ansible/pull/73493</a></li>\n<li><a href=\"https://github.com/ansible/ansible/pull/73494\">https://github.com/ansible/ansible/pull/73494</a></li>\n<li><a href=\"https://bugzilla.redhat.com/show_bug.cgi?id=1925002\">https://bugzilla.redhat.com/show_bug.cgi?id=1925002</a></li>\n<li><a href=\"https://www.debian.org/security/2021/dsa-4950\">https://www.debian.org/security/2021/dsa-4950</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-5rrg-rr89-x9mv\">https://github.com/advisories/GHSA-5rrg-rr89-x9mv</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-5rrg-rr89-x9mv",
            "title": "[ansible] Exposure of Sensitive Information in ansible",
            "date_modified": "2022-05-25T19:22:36.000Z",
            "date_published": "2022-05-25T19:22:34.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-5rrg-rr89-x9mv"
            }
        }
    ]
}